cfgov/unprocessed/apps/analytics-gtm/js/accessibility-interactions-listeners.js

```diff
@@ -0,0 +1,13 @@
+import {
+  addEventListenerToElem,
+  track
+} from './util/analytics-util';
+
+const AccessibleInteractionsListeners = ( function() {
```

Comment:
This should be updated to an arrow function to match below and our updated standards.

Reply:
Fixed